pkg/annotations/annotations.go

```diff
@@ -286,9 +286,11 @@ const (
 	// This should be used for implicit vNUMA topology.
 	NumaMaximumProcessorsPerNode = "io.microsoft.virtualmachine.computetopology.processor.numa.max-processors-per-node"
 
-	// NumaMaximumSizePerNode is the maximum size per vNUMA node.
+	// NumaMaximumMemorySizePerNode is the maximum memory size (in MB) per vNUMA node.
 	// This should be used for implicit vNUMA topology.
-	NumaMaximumSizePerNode = "io.microsoft.virtualmachine.computetopology.processor.numa.max-size-per-node"
+	NumaMaximumMemorySizePerNode = "io.microsoft.virtualmachine.computetopology.processor.numa.max-size-per-node"
+	// Deprecated: Use [NumaMaximumMemorySizePerNode] instead.
+	NumaMaximumSizePerNode = NumaMaximumMemorySizePerNode
```

Comment:
This is a breaking change due to the annotation rename. Since the code is internal for the defns you should just fix all code that used that naming. Or you need to keep the old tag while you go through deprecation.